client_generic/ContentDownloader/SheepGenerator.cpp

```diff
@@ -302,6 +302,13 @@ bool	SheepGenerator::getControlPoints( SheepUploader *uploader )
 																					1.0,
 																					1.0 );
 
+	// Convert "r3d0.sheepserver.net" to "v3d0.sheepserver.net" 
```

Comment:
no please do it like i said.  the above will not fix r3d1.sheepserver.net for example, should that come out next year.
just be sure it's defined in one place.